transformations/diacritic_removal/README.md

```diff
@@ -0,0 +1,16 @@
+# Diacritic Removal 
+This perturbation removes the diacritics on characters in a sentence.
+
+Author: Vikas Raunak (viraunak@microsoft.com)
+
+## What type of a transformation is this?
+This transformation removes the accented characters or diacritics on characters, and replaces them with their non-accented versions.
```

Comment:
Could you add an example of an accented character and a diacritic character?

Reply:
added the example of diacritic marks